sidecar-pg.Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM datamon-binaries as base
+
+# dist-alike during development/debug
+RUN cp /stage/usr/bin/datamon /usr/bin/datamon
+ADD ./hack/fuse-demo/datamon.yaml /root/.datamon/datamon.yaml
+
+# Build the dist image
+# ??? do not need fuse and sudo in case of bundle download instead of mount
```

Comment:
main question off top:  use `bundle mount` or `download` for existing bundles?

Reply:
I might have missed some nuances in datamon terminology, but "mount" would better correspond to "make the database available", while "download" would correspond to "make the database _storage_ available". Just saying.

Reply:
Mount is specifically used when the bundle is mounted as a filesystem (no data downloaded locally) vs download is where all the data is downloaded to the local node.